sdk/template/ci.yml

```diff
@@ -26,5 +26,5 @@ extends:
   parameters:
     ServiceDirectory: template
     Artifacts:
-      - name: azure-template
+      - name: @azure/template
```

Comment:
@praveenkuttappan testing this with template and if it works I think we should try and do this for all the artifact names so we can try and eliminate any confusion between package and artifact naming.